Packs/CortexCloudSec/Integrations/AWS/AWS_test.py

```diff
@@ -0,0 +1,192 @@
+import importlib
+import pytest
+
+AWS_INTEGRATION = importlib.import_module("AWS")
```

Comment:
```suggestion
from AWS import *
```
or import specific functions